types/sse.d.ts

```diff
@@ -0,0 +1,90 @@
+export type SSE = {
+    /**
+     * - headers
+     */
+    headers: string;
+    /**
+     * - payload as a string
+     */
+    payload: string;
+    /**
+     * - HTTP Method
+     */
+    method: string;
+    /**
+     * - flag, if credentials needed
+     */
+    withCredentials: boolean;
+    /**
+     * - debugging flag
+     */
+    debug: boolean;
```

Comment:
missing some fields of class
```suggestion
    debug: boolean;
    FIELD_SEPARATOR: string;
    listeners: Record<string, Function[]>;
    xhr: XMLHttpRequest | null;
    readyState: number;
    progress: number;
    chunk: string;
```